securedrop/sdconfig.py

```diff
@@ -11,154 +14,216 @@
     # http://flake8.pycqa.org/en/latest/user/error-codes.html?highlight=f401
     from typing import List, Dict  # noqa: F401
 
+CONFIG_DIR = '/etc/securedrop'
+
+
+class _FlaskConfig(object):
+
+    DEBUG = False
+    TESTING = False
+    WTF_CSRF_ENABLED = True
+
+    def __init__(self, secret_key, cookie_name, env):
+        # type: (str, str, str) -> None
+        self.SECRET_KEY = secret_key
+        self.SESSION_COOKIE_NAME = cookie_name
+
+        if env == 'prod':
+            # This is recommended for performance, and also resolves #369
+            self.USE_X_SENDFILE = True  # type: ignore
+        elif env == 'dev':
+            # Enable _Flask's debugger for development
+            self.DEBUG = True
+            # Use MAX_CONTENT_LENGTH to mimic the behavior of Apache's LimitRequestBody  # noqa: 501
+            # in the development environment. See #1714.
+            self.MAX_CONTENT_LENGTH = 524288000  # type: ignore
+        elif env == 'test':
+            self.TESTING = True
+            # Disable CSRF checks to make writing tests easier
+            self.WTF_CSRF_ENABLED = False
+
 
 class SDConfig(object):
-    def __init__(self):
-        # type: () -> None
-        try:
-            self.JournalistInterfaceFlaskConfig = \
-                _config.JournalistInterfaceFlaskConfig  # type: ignore
-        except AttributeError:
-            pass
 
-        try:
-            self.SourceInterfaceFlaskConfig = \
-                _config.SourceInterfaceFlaskConfig  # type: ignore
-        except AttributeError:
-            pass
+    def __init__(self, config_file):
+        # type: (str) -> None
 
-        try:
-            self.DATABASE_FILE = _config.DATABASE_FILE  # type: ignore
-        except AttributeError:
-            pass
+        with open(config_file) as f:
+            self._json_config = json.loads(f.read())
 
-        try:
-            self.DATABASE_USERNAME = _config.DATABASE_USERNAME  # type: ignore
-        except AttributeError:
-            pass
+        self.SECUREDROP_DATA_ROOT = '/var/lib/securedrop/'
```

Comment:
Please add the type annotations.

Reply:
Can you check again that sufficient type annotations have been added?